src/assets/style/Confirm.css

```diff
@@ -0,0 +1,20 @@
+.container {
+	border: 2px solid #0347b5;
+	width: 40%;
+	margin: 0 auto;
+	font-family: "Roboto", sans-serif;
+	border-radius: 2em;
+}
+
+.container > div {
+	text-align: center;
+	font-size: 2em;
+}
+.container > div:first-child {
+	margin-top: 1em;
+}
+.smol-text {
+	font-size: 1em !important;
```

Comment:
You should never use "!important"!!!😡

But I see why it is acceptable in this project.